package.json

```diff
@@ -71,7 +71,7 @@
     "react-plotly.js": "^2.5.1",
     "react-plotlyjs-ts": "^2.2.1",
     "react-popper": "^2.2.5",
-    "react-query": "^3.25.1",
+    "react-query": "3.31.0",
```

Comment:
The types in react-query are unstable in minor version releases so we need to pin a particular version. Ran into typescript issues in Agora because I was pulling a newer version of react-query than I expected